hatch.toml

```diff
@@ -29,9 +29,6 @@ overrides.matrix.deps.python = [
     { if = [ "min" ], value = "3.11" },
     { if = [ "stable", "pre" ], value = "3.13" },
 ]
-overrides.matrix.deps.features = [
-    { if = [ "stable", "pre" ], value = "test-full" },
-]
```

Comment:
why revert this?

Reply:
Simply because it was in the PR that was reverted.  I didn't think to check that it was its own "feature"